### DISCUSSION TITLE
jsonnet-bundler package names are confusing

### DISCUSSION THREAD

**Discussion by maxbrunet** [renovatebot/renovate#13529] [Suggest an Idea]
TL;DR I think package names should simply be repository with host + subdir (similar to `gomod`)

Currently the `jsonnet-bundler` manager tries to:

1. Use the `name` field. This is a legacy field ([jsonnet-bundler/jsonnet-bundler#63](https://github.com/jsonnet-bundler/jsonnet-bundler/pull/63)) which, in my opinion, it should not use
2. If `name` is not available, it falls back to the repository name in `remote` (e.g. `jsonnet-libs` for `"remote": "https://github.com/grafana/jsonnet-libs.git"`)
  * The repository name is not enough to identify a package
  * This is inconsistent with the legacy behavior mentioned in 1, which falls back to base name of the `subdir` if available, before falling back to the repository name

I think `jsonnet-bundler` aims to structure its packages similarly to how Golang does ([jsonnet-bundler/jsonnet-bundler#61](https://github.com/jsonnet-bundler/jsonnet-bundler/issues/61)), is it a refactor we could make in Renovate to be inline with this philosophy?

**Comment by viceice** [comment]
You can open a feature request and add all your findings and what needs to be done to fix / change current behavior.

**Reply by maxbrunet** [reply]
Opened #13568!
